python/ray/experimental/workflow/workflow_access.py

```diff
@@ -144,3 +192,14 @@ def report_success(self, workflow_id: str) -> None:
         """
         logger.info(f"Workflow job [id={workflow_id}] succeeded.")
         self._workflow_outputs.pop(workflow_id, None)
+
+
+def get_or_create_management_actor() -> "ActorHandle":
```

Comment:
Should we have a global variable for this? Otherwise, within the same py process, we'll always need to call ray.get_actor which is not very efficient I think?